src/core/wee-command.c

```diff
@@ -5189,6 +5189,22 @@ command_reload_file (struct t_config_file *config_file)
     }
 }
 
+/*
+ * Reloads all configuration files.
+ */
+
+void
+command_reload_files ()
```

Comment:
Perhaps this (and `command_reload_file`) belongs in `wee-config-file.c`?

Reply:
Yes, right, I'll move this function.